### DISCUSSION TITLE
Can't copy column names in Table Editor with cmd+c

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#16840] [Contribute to Supabase]
In the Table Editor, you're not able to copy column names with `cmd+c`

To reproduce: 
1. create a new table, with a column
2. select the column name
3. hit cmd+c
4. try to paste that value
5. find that it was never copied

Right click→ copy still works, but cmd+c should also work.

![CleanShot 2023-08-07 at 10 22 28@2x](https://github.com/supabase/supabase/assets/105593/b0c017a0-5e70-4c92-be32-51ebdf382e8e)

**Comment by faizanraso** [comment]
@saltcod How exactly were you copying? When i right click I get this - no option to copy:

<img width="1286" alt="Screenshot 2023-08-08 at 9 50 12 PM" src="https://github.com/supabase/supabase/assets/59986120/b5216a2d-645a-4575-8860-3b7a19bf47a4">

**Comment by saltcod** [comment]
This now works. Marking as resolved.
